run_dir/static/js/project_samples.js

```diff
@@ -468,6 +468,21 @@ function load_all_udfs(){
         $('#textarea_internal_costs').html(value);
       }
 
+      //Add connected projects if any
+      else if (key == 'project_xref'){
+        projs = '';
+        var conn_proj = value.split(',');
+        conn_proj.forEach(function(proj){
+           if (proj != project){
+              if (!projs.trim()){
+                 projs += '<span class="email_link"><a href="/project/'+proj+'">'+proj+'</a>'+'</span>';
```

Comment:
The `email_link` class can be put in the anchor tag like `<a class='email_link' href=..` and all the spans can be removed. You also don't need the `&nbsp;`.

Reply:
OK, great, less code! :-)